### PR TITLE
Add index.js to make it easier to consume this as a package.

### PR DESCRIPTION
**Pull request by nyergler**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const { debug, info, error } = require("./src/utils");
```
Comment (killercup) [inline index.js]: I'm pretty sure that's not necessary

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "hello-wasm",
+  "name": "wasmwrap",
```
Comment (killercup) [inline package.json]: Hm, if we decide on a name, we should probably do this a bit more consciously, and e.g. choose one that notes that this package is only meant to be used with Rust.

**Review by killercup — COMMENTED** [review]
Thanks! I'm not sure I really *want* this to be easy to consume right now, though: It's pretty much experimental and has some shortcomings…

But well, in the end, it probably doesn't matter much. So, let's do this, right here right now!

* Set the version to 0.0.1 -- this is in no way 1.0 worthy
* Decide on a name -- _wasmwrap_ is not bad, but sadly way too generic for what this does. How about _wrap-rust-wasm_? @badboy, any ideas/preferences?

**Comment by nyergler** [comment]
Thanks for the quick reply, @killercup. I get that it's experimental, just so happens I have an experimental rust-wasm project right now that your code made a lot easier :).

Agreed re: version, makes sense re: name; `wasm-rust`? `wasm-oxide`?

**Comment by killercup** [comment]
> name; wasm-rust? wasm-oxide?

Oh, I like the way you're thinking! Let's see if we can come up with a name that includes: coffee (it's Java-script after all), assembly, and oxidation?

**Comment by badboy** [comment]
[Hemileia vastatrix](https://en.wikipedia.org/wiki/Hemileia_vastatrix).

> Hemileia vastatrix is a fungus of the order Pucciniales (previously also known as Uredinales) that causes coffee leaf rust (CLR), ...

(yeah. not the nicest...)

**Comment by killercup** [comment]
Are German names still cool? Cause Wikipedia says Kaffeerost is a thing

Also: Woah you actually found a rust/fungus thing that works!

**Comment by nyergler** [comment]
Well it _is_ experimental, so something vaguely threatening like `kaffeerost` seems appropriate ;)

**Comment by killercup** [comment]
Kaffeerost it is, then! Can you amend the commit?

**Comment by nyergler** [comment]
@killercup Done!

**Comment by killercup** [comment]
Cheers!

bors r+

**Comment by badboy** [comment]
Great! Now onto making this a hybrid nodejs/web-env lib :P

**Comment by killercup** [comment]
Woah, the name kaffeerost is not yet taken on npm. I might even publish this 😮

**Comment by killercup** [comment]
@badboy you should probably use/contribute to https://github.com/browserify/rustify if you like browsers so much!

**Comment by killercup** [comment]
Oops, it happened: https://www.npmjs.com/package/kaffeerost

Thanks again, @nyergler!

**Pull request merged**